Projects.html

```diff
@@ -60,7 +60,7 @@ <h4>Lrem</h4>
             <a href="#"><img src=".\img\GitHub-Mark-32px.png" alt=""></a>
             <a href=""><img src=".\img\iconfinder_06-facebook_104498.png" alt=""></a>
             <a href=""> <img src=".\img\iconfinder_Instagram_1298747.png" alt=""></a>
-            <a href=""><img src=".\img\iconfinder_Linkedin_Color_1435161.png" alt=""></a>
+            <a href=""><img src="  .\img/iconfinder_square-linkedin_317725.png" alt=""></a>
```

Comment:
also missing an `alt` attribute here

Reply:
I did not add alt value in any img tag, i will fix that ASAP

Reply:
Fixed